src/components/locale-toggle/LocaleToggle.tsx

```diff
@@ -0,0 +1,44 @@
+import { useAtom } from 'jotai';
+import strings from '../../localization/strings';
+import { Button } from '@mui/material';
+import { languageAtom } from '../../atoms/languageAtom';
+import { Language } from '../../types';
+
+/**
+ * Function to change language of the app
+ */
+function LocaleToggle() {
+    const [language, setLanguage] = useAtom(languageAtom);
+
+    const handleLocaleChange = (newLanguage: string) => {
+        setLanguage(newLanguage as Language);
+        strings.setLanguage(newLanguage);
```

Comment:
I think this is already being taken care of within the languageAtom changes you have made, so this line strings.setLanguage(newLanguage);
Can be removed here